core/src/View/Kokkos_ViewMapping.hpp

```diff
@@ -3353,6 +3353,9 @@ KOKKOS_FUNCTION bool within_range(Map const& map,
   return (((std::size_t)indices < map.extent(Enumerate)) && ...);
 }
 
+// Disabled when using MDSpan because the MDSpan implementation has its own
+// version
+#ifndef KOKKOS_ENABLE_IMPL_MDSPAN
```

Comment:
This function looks the exact same as in `core/src/View/Kokkos_ViewAccessPreconditionsCheck.hpp`, could we just have this in one place?